libr/util/print.c

```diff
@@ -458,6 +459,24 @@ R_API void r_print_code(RPrint *p, ut64 addr, ut8 *buf, int len, char lang) {
 		}
 		p->cb_printf ("\n");
 		break;
+	case 'h':
+		{
+		ut16 *pbuf = (ut16*)buf;
+		w = 7;
+		ws = 2;
+		len /= ws;
+		p->cb_printf ("#define _BUFFER_SIZE %d\n", len);
+		p->cb_printf ("unsigned short buffer[%d] = {", len);
+		p->interrupt = 0;
+		for (i=0; !p->interrupt && i<len; i++) {
+			if (!(i%w)) p->cb_printf ("\n  ");
+			r_print_cursor (p, i, 1);
+			p->cb_printf ("0x%04x, ", pbuf[i]);
```

Comment:
See the code in latest pc. Current output looks better and this code is copypaste from there